### PR TITLE
Initial creation of members, maintainers and owners documents

### PR DESCRIPTION
**Pull request by dougbtv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +25,7 @@ meet in the same [Zoom channel](https://zoom.us/j/361123509) as is used for SIG-
 ## Working Items
 
 * Multi-Network CRD Specification
-  * [Repositories](https://github.com/K8sNetworkPlumbingWG/multi-net-spec): It contains finalized version.
+  * [Repositories](https://github.com/k8snetworkplumbingwg/community/issues): It contains finalized version.
```
Comment (Billy99) [inline README.md:28]: Not sure this change was on purpose. Were you jsut trying to change the case of the `/K8sNetworkPlumbingWG/`?

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Maintainers
```
Comment (Billy99) [inline MAINTAINERS.md:1]: While you are here, do you want to add the other repos: SR-IOV DP & CNI and device-info-spec?

File: README.md (hunk 1 above)
Comment (dougbtv) [reply README.md]: Thanks! Copy/paste error, thanks for catching it. Yeah, intended to match the changed case (from last year!)

File: MAINTAINERS.md (hunk 2 above)
Comment (dougbtv) [reply MAINTAINERS.md:1]: Good call! Thanks.

**Pull request merged**